packages/components/modal/src/modal.ts

```diff
@@ -69,6 +69,11 @@ export const modalProps = buildProps({
     required: false,
     default: undefined,
   },
+  hideHeader: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
```

Comment:
Hm... ce serait pas mieux de juste rendre le prop `title` optional ? Parce que la ca veut dire que je suis forcé de donner un title et apres de mettre `hideHeader` a true c'est pas folichon

Reply:
oh, nice catch indeed. the prop title is required.....
and that's KISS too.

Reply:
Indeed, seems better as you suggest